python/lsst/rubintv/production/utils.py

```diff
@@ -478,7 +478,7 @@ def nightReportPath(self):
     @cached_property
     def comCamButlerPath(self):
         file = self._config["comCamButlerPath"]
-        self._checkFile(file)
+        # self._checkFile(file)
```

Comment:
Might be better to just delete this.

Reply:
Yeah, I think Colin had to add that to add Sasquatch butler support. This whole class needs a total overhaul tbh, so sure, I've just deleted for now.